(window["webpackJsonp"]=window["webpackJsonp"]||[]).push([["chunk-4008d2ab"],{"023a":function(t,e,s){t.exports=s.p+"img/help_BulkPCA.8502e99e.svg"},"038a":function(t,e,s){t.exports=s.p+"img/help_eryAtlaSample.4bf05272.svg"},"0abb":function(t,e,s){t.exports=s.p+"img/help_scDifferTable.7b5e2e78.svg"},"0b8d":function(t,e,s){t.exports=s.p+"img/help_homeGenes.87bd514e.svg"},"12cd7":function(t,e,s){t.exports=s.p+"img/help_eryAtlas1.af862e67.svg"},"136a":function(t,e,s){t.exports=s.p+"img/help_about.a1f7a480.svg"},1385:function(t,e,s){t.exports=s.p+"img/help_homeDiffer.9efdfa75.svg"},1944:function(t,e,s){t.exports=s.p+"img/help_scDifferEnrich.8222a6a7.svg"},"1a23":function(t,e,s){t.exports=s.p+"img/help_homePage2.45108417.svg"},"1bfa":function(t,e,s){t.exports=s.p+"img/help_scDifferenceTra.091d1593.svg"},3108:function(t,e,s){"use strict";s("749c")},"3c02":function(t,e,s){t.exports=s.p+"img/help_expPro.18bd71a0.svg"},"410c":function(t,e,s){t.exports=s.p+"img/help_scCommuNetP.b8cb10f2.svg"},"469f":function(t,e,s){t.exports=s.p+"img/help_scDiffer.462275d6.svg"},"4de1":function(t,e,s){t.exports=s.p+"img/help_datasetAnalSC.df7c62f6.svg"},"5c74":function(t,e,s){t.exports=s.p+"img/help_scMarkerEnrich.b193129a.svg"},"6deb":function(t,e,s){t.exports=s.p+"img/help_scFeature.a68b4da1.svg"},"749c":function(t,e,s){},"77b2":function(t,e,s){t.exports=s.p+"img/help_BulkDiff.f4ddc20d.svg"},"7cef":function(t,e,s){t.exports=s.p+"img/help_homePage.7c5f1a51.svg"},"91d0":function(t,e,s){"use strict";s.r(e);var i=function(){var t=this,e=t.$createElement,s=t._self._c||e;return s("div",[s("div",[s("ul",{staticClass:"navs"},[s("button",{staticClass:"help_button",on:{click:function(e){t.isActive=!t.isActive}}},[s("Icon",{attrs:{type:"ios-arrow-down"}})],1),s("collapse",[s("div",{directives:[{name:"show",rawName:"v-show",value:t.isActive,expression:"isActive"}],staticClass:"container"},[s("li",[s("a",{staticClass:"a1",attrs:{href:"#Home"}},[t._v("Home")])]),s("li",[s("a",{attrs:{href:"#Differentiation"}},[t._v("  Differentiation")])]),s("li",[s("a",{attrs:{href:"#Genes"}},[t._v("  Genes")])]),s("li",[s("a",{attrs:{href:"#Compounds"}},[t._v("  Compounds")])]),s("li",[s("a",{attrs:{href:"#Disease"}},[t._v("  Disease")])]),s("li",[s("a",{attrs:{href:"#Search"}},[t._v("Search")])]),s("li",[s("a",{attrs:{href:"#Dataset"}},[t._v("Dataset")])]),s("li",[s("a",{attrs:{href:"#DatasetService"}},[t._v("Dataset Service")])]),s("li",[s("a",{attrs:{href:"#ExpressionProfile"}},[t._v("  Expression Profile")])]),s("li",[s("a",{attrs:{href:"#PrincipalComponents"}},[t._v("  Principal Components")])]),s("li",[s("a",{attrs:{href:"#DifferentialAnalysis"}},[t._v("  Differential Analysis")])]),s("li",[s("a",{attrs:{href:"#EnrichmentAnalysis"}},[t._v("  Enrichment Analysis")])]),s("li",[s("a",{attrs:{href:"#DRF"}},[t._v("  Dimensional Reduction & Feature")])]),s("li",[s("a",{attrs:{href:"#ME"}},[t._v("  Markers & Enrich")])]),s("li",[s("a",{attrs:{href:"#DE"}},[t._v("  Differential & Enrich")])]),s("li",[s("a",{attrs:{href:"#DT"}},[t._v("  Differentiation Trajectory")])]),s("li",[s("a",{attrs:{href:"#CCI"}},[t._v("  Cell-Cell Interraction")])]),s("li",[s("a",{attrs:{href:"#CCC"}},[t._v("  Cell-Cell Communication")])]),s("li",[s("a",{attrs:{href:"#Erythroid"}},[t._v("Erythroid Atlas")])]),s("li",[s("a",{attrs:{href:"#About"}},[t._v("About")])])])])],1)]),t._m(0)])},a=[function(){var t=this,e=t.$createElement,i=t._self._c||e;return i("div",{staticClass:"content"},[i("div",[i("h1",{staticClass:"h1_title"},[t._v(" Tutorial for Erythroid Database ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" Erythroid Database: A database of transcriptome profile of erythroid development. We developed the erythroid database EryDB,an open-access and comprehensive database dedicated to the collection, integration, analysis, and visualization of transcriptomic data for erythropoiesis and related diseases. Currently, the database includes expertly curated quality-assured data of 3,802 samples and 1,167,800 single cells derived from 94 public studies of three species (Homo sapiens, Mus musculus, and Danio rerio), nine tissue types, and seven diseases. EryDB provides the user with the ability to not only browse the molecular features of erythropoiesis between tissues and species, but also perform computational analyses of single-cell and bulk RNA-Seq data, thus serving as a convenient platform for custom queries and analyses. ")])]),i("div",{attrs:{id:"Home"}},[i("h1",{staticClass:"h1_title"},[t._v(" Home page ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" In the quik search box, you can search for genes(GATA1),tissues(Bone Marrow),Dataset ID(GSE102182) and species(homo sapiens). It will redirect you to the corresponding page(Search page) and you will get the datasets related to your input. The clickable 'About EryDB' on the top is a representative display of the simpe introduction of this database, and a click will easily hide it. ")]),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("7cef"),alt:"img"}})]),i("br"),i("br"),i("p",{staticClass:"p_text"},[t._v(" In the Home page you can also search datasets by four moudles. 1. By the cell type; 2. By the reported gene; 3. By the compound name; 4. By the disease type. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("1a23"),alt:"img"}})])]),i("div",{attrs:{id:"Differentiation"}},[i("h3",[i("li",[t._v("Differentiation")])]),i("br"),i("p",{staticClass:"p_text"},[t._v(" This moudle allows you to get datasets according to the celltype. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("1385"),alt:"img"}})])]),i("br"),i("br"),i("div",{attrs:{id:"Genes"}},[i("h3",[i("li",[t._v("Genes")])]),i("br"),i("p",{staticClass:"p_text"},[t._v(" This moudle allows you to get datasets according to the reported genes. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("0b8d"),alt:"img"}})]),i("br"),i("br")]),i("div",{attrs:{id:"Compounds"}},[i("h3",[i("li",[t._v("Compounds")])]),i("br"),i("p",{staticClass:"p_text"},[t._v(" This moudle allows you to get datasets according to the compounds. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("aa68"),alt:"img"}})])]),i("div",{attrs:{id:"Disease"}},[i("h3",[i("li",[t._v("Disease")])]),i("br"),i("p",{staticClass:"p_text"},[t._v(" This moudle allows you to get datasets according to the Diseases. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("b655"),alt:"img"}})])]),i("div",{attrs:{id:"Search"}},[i("h3",[t._v(" Search page ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" In this page, the advanced search is provided relative to the quick search. In specific, you can search for multiple aspects including cell type,reported gene,compound,speceis,tissue,experiment type,and omics type. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("c53a"),alt:"img"}})]),i("br"),i("br"),i("h1",{staticClass:"h1_title",attrs:{id:"Dataset"}},[t._v(" Dataset ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" When you get the interested dataset from searching,clicking the dataset(each row) will easily redirect you to the corresponding page(Dataset Service) ")]),i("br"),i("br"),i("h3",[i("li",[t._v("Dataset Details")])]),i("br"),i("p",{staticClass:"p_text"},[t._v(" This panel represents basic details of this study including the dataset id, organism, tissue, growth mode, development type , experiment type, PMID and so on. ")]),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("b9eb"),alt:"img"}})]),i("br"),i("br"),i("h3",[i("li",[t._v("Dataset Sample")])]),i("br"),i("p",{staticClass:"p_text"},[t._v(" This panel represents one or multiple samples contained in this dataset, including the information of species, tissue, and cell number.")]),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("c957"),alt:"img"}})]),i("h3",{attrs:{id:"DatasetService"}},[i("li",[t._v("Dataset Service")])]),i("br"),i("p",{staticClass:"p_text"},[t._v(" This panel represents analysis provided by this dataset. In general,datasets of bulk sequence provide the following analysis function 1)Expression Profile; 2)principal Compnents; 3)Differential Analysis; 4)Enrichment Analysis. ")]),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("ebe2d"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"ExpressionProfile"}},[t._v(" 1)Expression Profile ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" The barbox shows the number of expression level of gene in different group. You can input your interested gene through the input box in the upper left corner and get more information from linked wed. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("3c02"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"Principal Components"}},[t._v(" 2)Principal Components ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" The dot plot shows the Principal Compoment Analysis of samples in dataset. You can choose your interested group through clicking the dot in the right lable and get group information from dataset's sample information. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("023a"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"DifferentialAnalysis"}},[t._v(" 3)Differential Analysis ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" The dot plot shows the differential expressiona of samples in group. You can click the 'Contrasts Group' to select interested group. Here 'LMO2-GATA1' means the LMO2 group versus the GATA1 group. The signifcant differential genes is colored by orange and the unsignifcant differential gene is colored by bule. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("77b2"),alt:"img"}})]),i("br"),i("p",{staticClass:"p_text"},[t._v(" In the following table , you can search your interested gene by input the gene symbol. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("a552"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"EnrichmentAnalysis"}},[t._v(" 4)Enrichment Analysis ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" The bar plot shows the enrichment of the signifcant differential gene of group. You can click the 'Contrasts Group' to select interested group and click the 'Tpye' to select the interested enrichment type. The orange reprents the up group the the blue reprents the down group. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("dc98"),alt:"img"}})]),i("br"),i("br"),i("p",{staticClass:"p_text"},[t._v(" If this dataset was of single cell sequence,it provides the following analysis function. 1)Dimensional Reduction & Feature; 2)Markers & Enrich; 3)Differential & Enrich; 4)Differentiation Trajectory; 5)Cell-Cell Interaction; 6)Cell-Cell Communication. ")]),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("4de1"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"DRF"}},[t._v(" 1)Dimensional Reduction & Feature ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" The PCA,t-SNE or UMAP graph are presented for cell clusters in dataset. The original annotation of cell type in the study is used as reference for clustering results. You can customize the visualization chart. Plot size decides the dot size in this chart.View method provides 2D or 3D. Visaul Method includes PCA,t-SNE and UMAP. Group means the sample's group.The dot usually can by colored by Phase(Cells' Phase), Celltype or orig.ident(original ident). ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("b7f9"),alt:"img"}})]),i("br"),i("p",{staticClass:"p_text"},[t._v(" In Feature Plot you can input your interested gene symbol,the you will get the gene expression level in all cells in a choosed group. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("6deb"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"ME"}},[t._v(" 2)Markers & Enrich ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" This panel shows the top cell markers highly specifically expressed in this cell type (with significant high fold change between this cell type and others in relevant scRNA-seq datasets; FDR < 0.05). ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("9733"),alt:"img"}})]),i("br"),i("p",{staticClass:"p_text"},[t._v(" In Cluster's Enrichment Analysis, you can get the enrichment pathway of an interested cell cluster. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("5c74"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"DE"}},[t._v(" 3)Differential & Enrich; ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" The dot plot shows the differential expressiona of cells in different group. You can click the 'Contrasts Group' to select interested group. Here 'Macrophages-iPSC-VS-Macrophages_PB' means the Macrophages of iPSC group versus the Macrophages of PB group. The signifcant differential genes is colored by orange and the unsignifcant differential gene is colored by bule. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("469f"),alt:"img"}})]),i("br"),i("p",{staticClass:"p_text"},[t._v(" In the following table , you also can search your interested gene by input the gene symbol. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("0abb"),alt:"img"}})]),i("br"),i("br"),i("p",{staticClass:"p_text"},[t._v(" The bar plot shows the enrichment of the signifcant differential gene of cluster. You can click the 'Contrasts Group' to select interested group and click the 'Enrich Tpye' to select the interested enrichment type. The orange reprents the up group the the blue reprents the down group. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("1944"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"DT"}},[t._v(" 4)Differentiation Trajectory; ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" The dot plot shows the differentiation trajectory of cells in a choosed group. You can click the 'Group' to select interested group.Here the first dot plot is Pseudotime Trajectory。 ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("d014"),alt:"img"}})]),i("br"),i("p",{staticClass:"p_text"},[t._v(" In the following table , you also can choose your interested group and color the dot by celltype,phanse or orig.ident. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("1bfa"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"CCI"}},[t._v(" 5)Cell-Cell Interraction; ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" The plot shows the interaction fact of cell cluster in a choosed group. You can click the 'Group' to select interested group and click the 'Communication type'(all, growth gactor,other,cytokie,checkpoint) to define the interrection type. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("af9c"),alt:"img"}})]),i("br"),i("br"),i("h4",{attrs:{id:"CCC"}},[t._v(" 6)Cell-Cell Communication. ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" The plot shows the communication pattern of cell cluster in a choosed group. You can click the 'Group' to select interested group and signal pattern(incoming and outgoing). ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("bf6d"),alt:"img"}})]),i("br"),i("br"),i("p",{staticClass:"p_text"},[t._v(" In the following plot , you can choose your interested signal name and check the signal name and the contribution of each L-R pair.. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("410c"),alt:"img"}})]),i("br")]),i("div",{attrs:{id:"Erythroid"}},[i("br"),i("br"),i("h3",[t._v(" Erythroid Atlas ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" In this page, we provide intergrated datasets for users to query their interested analysis. In specific, you can the choose organism and sequence type to see the intergrated datasets,which provide the same analysis content as a single dataset. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("bff3"),alt:"img"}})]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("12cd7"),alt:"img"}})]),i("br"),i("br"),i("p",{staticClass:"p_text"},[t._v(" In the bottom of this page, we provide the information of samples and groups about the intergrated datasets. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("038a"),alt:"img"}})])]),i("div",{attrs:{id:"About"}},[i("br"),i("br"),i("h3",[t._v(" About ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" In this page, we provide the statistic informaton about all the datasets. You can the datasets' number, samples' number ,and cells' number showed by organism, growth mode and tissue. In future releases of EryDB, transcriptomic data will be continuously updated, and more types of omics data will be added, including epigenomic, proteomic, metabolomic, and genomic data, as well as next-generation omics data types. ")]),i("br"),i("br"),i("p",{staticClass:"img_c"},[i("img",{attrs:{width:"80%",src:s("136a"),alt:"img"}})]),i("br"),i("br"),i("h3",[t._v(" About us ")]),i("br"),i("p",{staticClass:"p_text"},[t._v(" If you have any questions or comments, please feel free to contact us via email (zhenggm@big.ac.cn, fangxd@big.ac.cn). ")]),i("br")])])}],r="0.3s height ease-in-out, 0.3s padding-top ease-in-out, 0.3s padding-bottom ease-in-out",o={"before-enter":function(t){t.style.transition=r,t.dataset||(t.dataset={}),t.dataset.oldPaddingTop=t.style.paddingTop,t.dataset.oldPaddingBottom=t.style.paddingBottom,t.style.height=0,t.style.paddingTop=0,t.style.paddingBottom=0},enter:function(t){t.dataset.oldOverflow=t.style.overflow,0!==t.scrollHeight?(t.style.height=t.scrollHeight+"px",t.style.paddingTop=t.dataset.oldPaddingTop,t.style.paddingBottom=t.dataset.oldPaddingBottom):(t.style.height="",t.style.paddingTop=t.dataset.oldPaddingTop,t.style.paddingBottom=t.dataset.oldPaddingBottom),t.style.overflow="hidden"},"after-enter":function(t){t.style.transition="",t.style.height="",t.style.overflow=t.dataset.oldOverflow},"before-leave":function(t){t.dataset||(t.dataset={}),t.dataset.oldPaddingTop=t.style.paddingTop,t.dataset.oldPaddingBottom=t.style.paddingBottom,t.dataset.oldOverflow=t.style.overflow,t.style.height=t.scrollHeight+"pcontainerx",t.style.overflow="hidden"},leave:function(t){0!==t.scrollHeight&&(t.style.transition=r,t.style.height=0,t.style.paddingTop=0,t.style.paddingBottom=0)},"after-leave":function(t){t.style.transition="",t.style.height="",t.style.overflow=t.dataset.oldOverflow,t.style.paddingTop=t.dataset.oldPaddingTop,t.style.paddingBottom=t.dataset.oldPaddingBottom}},n={name:"collapseTransition",functional:!0,render:function(t,e){var s=e.children,i={on:o};return t("transition",i,s)}},l={name:"help",components:{collapse:n},data:function(){return{isActive:!0}},mounted:function(){},destroy:function(){},methods:{}},c=l,p=(s("3108"),s("2877")),d=Object(p["a"])(c,i,a,!1,null,null,null),h=d.exports;e["default"]=h},9733:function(t,e,s){t.exports=s.p+"img/help_scMarker.2b754afe.svg"},a552:function(t,e,s){t.exports=s.p+"img/help_BulkDiffTable.2ae95dc7.svg"},aa68:function(t,e,s){t.exports=s.p+"img/help_homeCompounds.18ed1e8d.svg"},af9c:function(t,e,s){t.exports=s.p+"img/help_scInter.60fb1759.svg"},b655:function(t,e,s){t.exports=s.p+"img/help_homeDiseases.32e8d02a.svg"},b7f9:function(t,e,s){t.exports=s.p+"img/help_scPCA.4ad41c0f.svg"},b9eb:function(t,e,s){t.exports=s.p+"img/help_datasetDetial.1f55eb78.svg"},bf6d:function(t,e,s){t.exports=s.p+"img/help_scCommu.54f18ccd.svg"},bff3:function(t,e,s){t.exports=s.p+"img/help_eryAtlas.9ecea367.svg"},c53a:function(t,e,s){t.exports=s.p+"img/help_search.01642066.svg"},c957:function(t,e,s){t.exports=s.p+"img/help_datasetSample.1ed05b6a.svg"},d014:function(t,e,s){t.exports=s.p+"img/help_scPseu.5342dfa4.svg"},dc98:function(t,e,s){t.exports=s.p+"img/help_BulkEnrich.80af17f2.svg"},ebe2d:function(t,e,s){t.exports=s.p+"img/help_datasetAnalBulk.78c40fbe.svg"}}]);